cmd/rpcdaemon/commands/daemon.go

```diff
@@ -86,6 +89,125 @@ func (api *APIImpl) BlockNumber(ctx context.Context) (hexutil.Uint64, error) {
 	return hexutil.Uint64(blockNumber), nil
 }
 
+func GetReceipts(ctx context.Context, db rawdb.DatabaseReader, cfg *params.ChainConfig, hash common.Hash) (types.Receipts, error) {
+	number := rawdb.ReadHeaderNumber(db, hash)
+	if number == nil {
+		return nil, nil
```

Comment:
Shall we return an error here? The same for different places below.

Reply:
My vote - yes.